notebooks/Testgrid_flakiness_detection.ipynb

```diff
@@ -0,0 +1,2600 @@
+{
```

Comment:
<p>Why not include Flip Flakiness with optimal distance here as well? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/37/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
<p>Done. Added</p>